app/assets/stylesheets/welcome.scss

```diff
@@ -1,158 +0,0 @@
-// Place all the styles related to the welcome controller here.
-// They will automatically be included in application.css.
-// You can use Sass (SCSS) here: http://sass-lang.com/
-@import "bootstrap-sprockets";
-@import "bootstrap";
-
-@font-face {
-  font-family: 'Myriad Pro';
-  src: asset-url('mp.woff');
-}
-
-$main-color: #f5c85f;
-$difuminated-color-border: #fff;
-
-.line {
-  background-color: $main-color;
-  height: 8px;
-  width: 100%;
-  margin-bottom: 40px;
-}
-
-.main-container {
-  display: flex;
-  justify-content: center;
-  flex-wrap: wrap;
-}
-
-.main-section {
-  display: flex;
-  justify-content: space-around;
-  flex-wrap: wrap;
-  vertical-align: top;
-  margin-bottom: 40px;
-  width: 500px;
-}
-
-.vertical-line {
-  height: 300px;
-  width: 2px;
-  background-size: cover;
-}
-
-.gradient {
-  background: $main-color;
-  background: -webkit-radial-gradient(circle, $main-color, $main-color, $difuminated-color-border);
-  background: -o-radial-gradient(circle, $main-color, $main-color, $difuminated-color-border);
-  background: -moz-radial-gradient(circle, $main-color, $main-color, $difuminated-color-border);
-  background: radial-gradient(circle, $main-color, $main-color, $difuminated-color-border);
-}
-
-.logo {
-  height: 100px;
-  width: 200px;
-  margin: auto 0;
-}
-
-.triangle-section {
-  background: url('/assets/triangle.png') right center no-repeat;
-  background-repeat: repeat-x;
-  height: 12px;
-}
-
-.container {
-  width: 100%;
-}
-
-.steps-section {
-  display: flex;
-  justify-content: center;
-  flex-wrap: wrap;
-  background: #eee;
-  padding-top: 60px;
-  padding-bottom: 60px;
-  -webkit-box-shadow: inset 0px -15px 44px 0px rgba(201,195,201,1);
-  -moz-box-shadow: inset 0px -15px 44px 0px rgba(201,195,201,1);
-  box-shadow: inset 0px -15px 44px 0px rgba(201,195,201,1);
-}
-
-.step {
-  text-align:center;
-  width: 230px;
-  margin: 20px;
-}
-
-.title {
-  font-family: 'Myriad Pro';
-  font-size: 19px;
-  color: #999;
-  line-height: 23px;
-  margin-bottom: 10px;
-}
-
-.subtitle {
-  font-family: 'Myriad Pro';
```

Comment:
font-family should be applied at a really high level to avoid repeating it in every selector.
```scss
* {
    font-family: 'Myriad Pro';
}